src/Applications/LDAS_App/GEOSldas_HIST.rc

```diff
@@ -17,6 +17,7 @@ COLLECTIONS:
 #CUBE            'tavg24_2d_lnd_Nx'
 #ASSIM           'SMAP_L4_SM_gph'
 #                'catch_progn_incr'
+#                'catch_diag_aup'
```

Comment:
We still need to clean up HISTORY conventions...
One concern is that it's probably impossible to keep the Collection label to three letters, but we can still indicated whether this is tavg or incr, what the time step is, and whether the output is in tile space.
For this new collection I suggest using:   "inst3_1d_[LABEL]_Nt" for instantaneous, 3-hourly, 1-dim increments output in tile space.
For label, perhaps use "lndfcstana"?  It's a bit misleading because we're not putting out everything that is in the "lnd" collection, although anything that would be in a inst3 lnd collection other than the 5 variables included here would represent both fcst and ana values (which are identical for our typical analysis use case).